frontend/src/components/Loader/index.tsx

```diff
@@ -1,30 +1,34 @@
 import React from "react";
-import { Box, CircularProgress, Typography } from "@mui/material";
+import { Box, CircularProgress, Typography, styled } from "@mui/material";
 
 type LoadingProps = {
   size?: number;
   message?: string;
 };
 
-const Loading: React.FC<LoadingProps> = ({ size, message }) => {
+const StyledBox = styled(Box)(() => ({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  flexDirection: "column",
+  height: "100vh",
+}));
+
+const StyledTypography = styled(Typography)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+}));
+
```

Comment:
we can add these styled box and typography inside the `styles.ts` and import from there

Reply:
Provide the example when you are reviewing

Reply:
It's simple import-export thing! I don't think any examples are required as i have clearly defined what to do above!!!